src/kvstore/plugins/hbase/HBaseStore.h

```diff
@@ -148,6 +155,15 @@ class HBaseStore : public KVStore {
         return ResultCode::ERR_UNSUPPORTED;
     }
 
+    ResultCode createSnapshot(GraphSpaceID) override {
+        return ResultCode::ERR_UNSUPPORTED;
```

Comment:
`LOG(FATAL) << "Not Implement"` is more reasonable :)

Reply:
I don't think so，there's no need to crash because of an error . and code needs to be uniform.